features/flags.json

```diff
@@ -246,6 +246,21 @@
       "enabled": true,
       "lastEditedBy": "Patrick Tajima",
       "lastEditedAt": "2022-05-06T17:30:49.430Z"
+    },
+    "testnet_NEARORG": {
+      "enabled": false,
```

Comment:
This should be `true` to match `testnet`

Reply:
great catch! thanks 👍